packages/manager/modules/dbaas-logs/src/logs/order/logs-order.html

```diff
@@ -0,0 +1,184 @@
+<div class="container">
+    <div class="row justify-content-center p-4 m-4">
+        <oui-spinner data-size="m" data-ng-if="ctrl.isLoading"></oui-spinner>
+    </div>
+    <ovh-manager-product-offers
+        ng-if="ctrl.workflowOptions.catalog && !ctrl.isLoading"
+        workflow-type="'order'"
+        workflow-options=":: ctrl.workflowOptions"
+        pricing-type="'renew'"
+        user=":: ctrl.user"
+        on-error=":: ctrl.onOrderFail(error)"
+        on-success=":: ctrl.onOrderSucceed(checkout)"
+        send-current-state=":: ctrl.getOrderState(state)"
+    >
+        <oui-step-form
+            name="logs_order_selection"
+            header="{{::'logs_order_selection_title' | translate}}"
+            description="{{::'logs_order_selection_subtitle' | translate}}"
+            valid="ctrl.selectedOffer"
+            on-submit="ctrl.onSubmitOffer"
+        >
+            <div class="row">
+                <oui-select-picker
+                    class="mb-3 col-md-5"
+                    name="logsStandardOffer"
+                    model="ctrl.selectedOffer"
+                    label="{{::'logs_order_selection_details_standard_title' | translate}}"
+                    values="['logs-account']"
+                    on-change="ctrl.selectOffer('logs-account')"
+                >
+                    <oui-select-picker-section>
+                        {{::'logs_order_selection_details_standard_description'
+                        | translate}}
+                        <ul>
+                            <li>
+                                {{::'logs_order_selection_details_standard_description_1'
+                                | translate}}
+                            </li>
+                            <li>
+                                {{::'logs_order_selection_details_standard_description_2'
+                                | translate}}
+                            </li>
+                            <li>
+                                {{::'logs_order_selection_details_standard_description_3'
+                                | translate}}
+                            </li>
+                            <li>
+                                {{::'logs_order_selection_details_standard_description_4'
+                                | translate}}
+                            </li>
+                            <li>
+                                {{::'logs_order_selection_details_standard_description_5'
+                                | translate}}
+                            </li>
+                        </ul>
+                    </oui-select-picker-section>
+                    <oui-select-picker-footer>
+                        <p
+                            class="font-weight-bold d-block m-0"
```

Comment:
```suggestion
                            class="font-weight-bold m-0"
```
this is default `p` element display 
Also is overriding the margin necessary ?

Reply:
`d-block` is unecessary yes but 'p' have a strong margin at bottom by default in bootstrap and it break display